client/push/push_manager.ts

```diff
@@ -67,6 +67,34 @@ export default class PushManager {
 
   async unregister(): Promise<boolean> {
     // Not implemented yet
+    if (navigator.serviceWorker) {
+      navigator.serviceWorker.register('/push_service_worker.js')
```

Comment:
This is unregister() function but why are u registering service worker here again?